README.md

```diff
@@ -130,42 +130,42 @@ easiest way to manage multiple node versions side by side. See
 
 ### Install dependencies
 
-Install dependencies with [yarn](https://yarnpkg.com/lang/en/docs/install/):
+Install dependencies with npm:
 
-```
-yarn install
+```sh
+npm install
 ```
 
 Dependencies are automatically kept up-to-date using [greenkeeper](http://greenkeeper.io/).
 
-### yarn scripts and grunt tasks
+### npm scripts and grunt tasks
 
-Run basic automation tasks via yarn (e.g. `yarn test`).
+Run basic automation tasks via npm (e.g. `npm test`).
 These don't need `grunt-cli` installed globally.
 
-#### yarn scripts
+#### npm scripts
 
-Most of these scripts will also run with npm by substuting `yarn` with `npm`.
+These scripts will also run with yarn by substuting `npm` with `yarn`.
```

Comment:
Not sure if we still want this. @EnTeQuAk what do you think? I am always wary of stuff that we tell works but don't test.